gradle/build-logic/convention/src/main/java/app/tivi/gradle/utils/Android.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.gradle.utils
```

Comment:
Would love to have a better name for the package 🙂 Thoughts?

Reply:
`helpers` maybe

Reply:
Utils seems fine, but just `app.tivi.gradle` could work too